src/transformStateWithClones.js

```diff
@@ -7,7 +7,48 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const cloneState = [];
+
+  let stateCopy = {
+    ...state,
+  };
```

Comment:
you can write in one line
```suggestion
  let stateCopy = { ...state };
```